leptos_dom/src/helpers.rs

```diff
@@ -134,10 +148,95 @@ pub fn set_timeout(cb: impl FnOnce() + 'static, duration: Duration) {
     }
 
     let cb = Closure::once_into_js(Box::new(cb) as Box<dyn FnOnce()>);
-    _ = window().set_timeout_with_callback_and_timeout_and_arguments_0(
-        cb.as_ref().unchecked_ref(),
-        duration.as_millis().try_into().unwrap_throw(),
-    );
+    window()
+        .set_timeout_with_callback_and_timeout_and_arguments_0(
+            cb.as_ref().unchecked_ref(),
+            duration.as_millis().try_into().unwrap_throw(),
+        )
+        .map(TimeoutHandle)
+}
+
+/// Handle that is generated by [set_interval] and can be used to clear the interval.
```

Comment:
This comment looks like it should be updated. Maybe to something like:
```
/// Handle that is generated by [set_timeout_with_handle] and can be used to clear the timeout.
```

Reply:
Ah yes of course, thanks!